### PR TITLE
[Android] TW-1696: Change app icon

### PR DESCRIPTION
**Pull request by hieutbui**
## Ticket
- #1696 

## DOD
- [x] icon
- [x] splash screen
- [x] noti icon
- [x] icon in app 

## Resolved
- Android 28:

https://github.com/linagora/twake-on-matrix/assets/80142234/b5647fc7-314d-4e4e-8639-fecb82b2fc9c

- Android 34:

https://github.com/linagora/twake-on-matrix/assets/80142234/9257c2c6-e188-4265-bf59-5cdc72d084f4

### CODE REVIEW

**Review by sherlockvn — APPROVED** [review]

File: android/app/src/main/res/drawable/launch_background.xml [hunk 1]
```diff
@@ -7,6 +7,6 @@
         <bitmap android:gravity="center" android:src="@drawable/splash"/>
     </item>
     <item>
-        <bitmap android:gravity="bottom" android:src="@drawable/branding" android:bottom="32dp"/>
```
Comment (hoangdat) [inline android/app/src/main/res/drawable/launch_background.xml:10]: no need to move up a bit?

**Comment by hoangdat** [comment]
- can you show me the app icon while app in recent list 
- show app icon in settings

**Comment by hieutbui** [comment]
> * can you show me the app icon while app in recent list
> * show app icon in settings

- Recent list:
image
recent list

- Settings:
app info

**Review by hoangdat — APPROVED** [review]

File: android/app/src/main/res/drawable/launch_background.xml (hunk 1 above)
Comment (hieutbui) [reply android/app/src/main/res/drawable/launch_background.xml:10]: I removed it cause it doesn't have any effects at all, and `<bitmap>`  also doesn't have attribute `android:bottom`.
RF:
- https://developer.android.com/reference/android/graphics/drawable/BitmapDrawable#summary
- https://developer.android.com/guide/topics/resources/drawable-resource#XmlBitmap

If we still need to move the branding up a bit from the bottom, the code should be:
```
<item android:bottom="32dp">
    <bitmap android:gravity="bottom" android:src="@drawable/branding"/>
</item>
```

**Pull request merged**